ush/setup.py

```diff
@@ -83,42 +82,64 @@ def setup():
     #
     EXPT_DEFAULT_CONFIG_FN = "config_defaults.yaml"
     cfg_d = load_config_file(os.path.join(USHdir, EXPT_DEFAULT_CONFIG_FN))
-    import_vars(dictionary=flatten_dict(cfg_d),
-        env_vars=["EXPT_CONFIG_FN",
-                  "EXTRN_MDL_NAME_ICS", "EXTRN_MDL_NAME_LBCS",
-                  "FV3GFS_FILE_FMT_ICS", "FV3GFS_FILE_FMT_LBCS"])
-
+    import_vars(
+        dictionary=flatten_dict(cfg_d),
+        env_vars=[
+            "EXPT_CONFIG_FN",
+            "EXTRN_MDL_NAME_ICS",
+            "EXTRN_MDL_NAME_LBCS",
+            "FV3GFS_FILE_FMT_ICS",
+            "FV3GFS_FILE_FMT_LBCS",
+        ],
+    )
 
     # Load the user config file, then ensure all user-specified
-    # variables correspond to a default value. 
+    # variables correspond to a default value.
     if not os.path.exists(EXPT_CONFIG_FN):
-        raise FileNotFoundError(f'User config file not found: EXPT_CONFIG_FN = {EXPT_CONFIG_FN}')
+        raise FileNotFoundError(
+            f"User config file not found: EXPT_CONFIG_FN = {EXPT_CONFIG_FN}"
+        )
 
     try:
         cfg_u = load_config_file(os.path.join(USHdir, EXPT_CONFIG_FN))
     except:
-        errmsg = dedent(f'''\n
-                        Could not load YAML config file:  {EXPT_CONFIG_FN}
-                        Reference the above traceback for more information.
-                        ''')
+        errmsg = dedent(
+            f"""\n
+            Could not load YAML config file:  {EXPT_CONFIG_FN}
+            Reference the above traceback for more information.
+            """
+        )
         raise Exception(errmsg)
 
     cfg_u = flatten_dict(cfg_u)
     for key in cfg_u:
         if key not in flatten_dict(cfg_d):
-            raise Exception(dedent(f'''
-                            User-specified variable "{key}" in {EXPT_CONFIG_FN} is not valid.
-                            Check {EXPT_DEFAULT_CONFIG_FN} for allowed user-specified variables.\n'''))
+            raise Exception(
+                dedent(
+                    f"""
+                    User-specified variable "{key}" in {EXPT_CONFIG_FN} is not valid.
+                    Check {EXPT_DEFAULT_CONFIG_FN} for allowed user-specified variables.\n"""
+                )
+            )
 
     # Mandatory variables *must* be set in the user's config; the default value is invalid
-    mandatory = ['MACHINE']
+    mandatory = ["MACHINE"]
     for val in mandatory:
         if val not in cfg_u:
-            raise Exception(f'Mandatory variable "{val}" not found in user config file {EXPT_CONFIG_FN}')
+            raise Exception(
+                f'Mandatory variable "{val}" not found in user config file {EXPT_CONFIG_FN}'
```

Comment:
For consistency, would it be better to convert the single quotes with double quotes, then convert the double quotes around `{val}` to single quotes?